assets/pages/oneoff-contributions/components/oneoffContributionsPayment.jsx

```diff
@@ -94,8 +124,11 @@ function OneoffContributionsPayment(props: PropTypes, context) {
           props.referrerAcquisitionData,
           context.store.getState,
         )}
-        canOpen={() => props.formFields.every(f => f.isValid)}
-        onClick={() => props.setShouldValidateFunctions.forEach(f => f())}
```

Comment:
This seems suspicious. Instead, the button should just be disabled if at least one validation fails.